integration/spark-common-test/src/test/resources/IUD/2000_UniqData_2.csv

```diff
@@ -0,0 +1,5 @@
+CUST_ID,CUST_NAME,DECIMAL_COLUMN1
```

Comment:
can you simulate the data for test, don't suggest adding many csv files to repo.